source/03-specifications/02-formats-for-counter-reports.rst

```diff
@@ -123,8 +123,8 @@ Table 3.f (below): COUNTER Report Header Elements
        360 COUNTER
 
    * - Registry_Record
-     - The link to the platform's COUNTER Registry record. Report providers who do not have a Registry record MUST leave the value blank.
-     - https://registry.projectcounter.org/platform/b2b2736c-2cb9-48ec-91f4-870336acfb1c
+     - The link to the platform's COUNTER Registry record. Report providers who do not have a Registry record MUST leave the value blank. Where a COUNTER Report is downloaded that includes usage data for multiple platforms, the Registry link should be to the Usage Data Host.
```

Comment:
Maybe we should say "multiple platforms in the COUNTER Registry" to make clear that this is not about the Platform element? There is quite a number of report providers which have just a single platform entry in the COUNTER Registry but multiple Platforms in the report.